app/src/main/java/org/oppia/app/help/faq/faqsingle/FAQSingleActivityPresenter.kt

```diff
@@ -37,12 +40,19 @@ class FAQSingleActivityPresenter @Inject constructor(
       (activity as FAQSingleActivity).finish()
     }
     activity.findViewById<TextView>(R.id.faq_question_text_view).text = question
-    activity.findViewById<TextView>(R.id.faq_answer_text_view).text = if (
-      Build.VERSION.SDK_INT >= Build.VERSION_CODES.N
-    ) {
-      Html.fromHtml(answer, Html.FROM_HTML_MODE_COMPACT)
-    } else {
-      Html.fromHtml(answer)
-    }
+
+    // NOTE: Here entityType and entityId can be anything as it will actually not get used.
+    // They are needed only for cases where rich-text contains images from server and in faq
+    // we do not have images.
+    val answerTextView = activity.findViewById<TextView>(R.id.faq_answer_text_view)
+    answerTextView.text = htmlParserFactory.create(
+      resourceBucketName,
+      /* entityType= */ "faq",
```

Comment:
as per our earlier discussion, should we go with named argument or this comment argument is fine?

Reply:
Done.